store/populations_test.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package store
+
+import (
+	"context"
+	"testing"
+
+	pb "github.com/datacommonsorg/mixer/proto"
+	"github.com/datacommonsorg/mixer/util"
+)
+
+func TestGetPopObs(t *testing.T) {
+	data := map[string]string{}
+	dcid := "geoId/06"
+	key := util.BtPopObsPrefix + dcid
+	btRow := `{
+	"name": "Santa Clara",
+  "populations": {
```

Comment:
Nit, is there any tool we could use to format those?

Reply:
I use visual studio code and golang formatter plugin. It usually does good job. In this case as it is a string, so seems dumb. I updated manually.